### PR TITLE
update the rest of dependencies 

### PR DESCRIPTION
**Pull request by jack60612**
please squash this on merge

### CODE REVIEW

**Comment by jack60612** [comment]
@seeden im ready for review

**Comment by jack60612** [comment]
@seeden every dependency is updated now lol

**Comment by seeden** [comment]
Thanks, I will check it tomorrow morning

File: src/electron/main.tsx
Comment (seeden) [inline src/electron/main.tsx:135]: can you enable contextIsolation?

File: src/electron/main.tsx
Comment (seeden) [inline src/electron/main.tsx:217]: please add reply for this function as well

**Review by seeden — CHANGES_REQUESTED** [review]
I have few comments but it looks good

File: src/electron/main.tsx
Comment (jack60612) [reply src/electron/main.tsx:135]: If context isolation is enabled then node intergation is disabled

Comment (jack60612) [reply src/electron/main.tsx:135]: Which is definitely needed

Comment (seeden) [reply src/electron/main.tsx:135]: are we able to do that in a different way without disabling contextIsolation?

Comment (jack60612) [reply src/electron/main.tsx:135]: I believe so but I need to look into it

File: src/electron/main.tsx
Comment (jack60612) [reply src/electron/main.tsx]: fixed

File: src/electron/main.tsx
Comment (jack60612) [reply src/electron/main.tsx:135]: > are we able to do that in a different way without disabling contextIsolation?

it will take me a bit

Comment (jack60612) [reply src/electron/main.tsx:135]: > are we able to do that in a different way without disabling contextIsolation?

we are not able to as enabling context isolation disables node integration (import) for the main renderer.
It is still more secure then before as remote is restricted

Comment (jack60612) [reply src/electron/main.tsx:135]: @seeden let me know if that answers your questions

Comment (seeden) [reply src/electron/main.tsx:135]: we can use contextBridge https://www.electronjs.org/docs/tutorial/context-isolation#migration

`
const { contextBridge } = require('electron')

contextBridge.exposeInMainWorld('myAPI', {
  doAThing: () => {}
})
`

Comment (jack60612) [reply src/electron/main.tsx:135]: Yep  i have already tried that,

Comment (jack60612) [reply src/electron/main.tsx:135]: > Yep i have already tried that,

i have it stashed but every file in the whole project will need to get its imports removed.
Its simply not feasible

Comment (jack60612) [reply src/electron/main.tsx:135]: Let me make a new branch with it

Comment (seeden) [reply src/electron/main.tsx:135]: ok

Comment (jack60612) [reply src/electron/main.tsx:135]: @seeden https://github.com/jack60612/chia-blockchain-gui/tree/testcontext

**Comment by seeden** [comment]
I will merge it next week with contextIsolation. 
Thank you

**Comment by jack60612** [comment]
@seeden do you want me to fix the conflicts?

**Comment by seeden** [comment]
Hi @jack60612 . Yes, please :) I have a lot of work with API refactoring and other things

**Comment by jack60612** [comment]
> Hi @jack60612 . Yes, please :) I have a lot of work with API refactoring and other things

All fixed @seeden

**Comment by jack60612** [comment]
> Hi @jack60612 . Yes, please :) I have a lot of work with API refactoring and other things

let me know if i could lend a hand.

**Comment by jack60612** [comment]
> Hi @jack60612 . Yes, please :) I have a lot of work with API refactoring and other things
@seeden please commit this so i can start working on more changes

**Comment by seeden** [comment]
Hi @jack60612 I was waiting for release. I will merge it today. Please take a look at branch big_split. I am working on it right now and the structure of the whole project will be changed. 
Maybe you should wait for the merge of this branch as well. Because conflicts will be huge

**Comment by jack60612** [comment]
Ok I will wait for big split to be done before doing any more commits but I might help out on the plotter selector thing

**Comment by jack60612** [comment]
> Hi @jack60612 I was waiting for release. I will merge it today. Please take a look at branch big_split. I am working on it right now and the structure of the whole project will be changed. Maybe you should wait for the merge of this branch as well. Because conflicts will be huge

if you want i'll fix the conflicts that this creates for big split

**Review by seeden — APPROVED** [review]

**Comment by seeden** [comment]
can you fix the current conflicts and I will merge it after that

**Comment by jack60612** [comment]
@seeden there are no conflicts

**Pull request merged**
